lib/github/ldap.rb

```diff
@@ -65,6 +65,7 @@ class Ldap
     #   which to attempt opening connections (default [[host, port]]). Overrides
     #   host and port if set.
     # encryption: optional string. `ssl` or `tls`. nil by default
+    # validate_encryption: optional boolean. nil by default
```

Comment:
@jonabc How about passing down `tls_options` itself? That gives maximum control over the specific options and also would make it possible to pass down specifics like a certificate store if ever needed.